srcpkgs/gifski/template

```diff
@@ -10,7 +10,11 @@ maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="AGPL-3.0-only"
 homepage="https://gif.ski"
 distfiles="https://github.com/ImageOptim/gifski/archive/${version}.tar.gz"
-checksum=4894f70c351ce8440c9cdaca30f3bd59630dc10ce4bb60bc7686fe3f4bcf641c
+checksum=f9f31cdc2d1670397b432e243aea4a3a35c25b9484fe256998a7dbc4bdac9b21
+
+pre_build() {
+	cargo update --package imagequant-sys
+}
```

Comment:
There was a bump in https://github.com/ImageOptim/gifski/commit/c722234a5a1dd113b81f0853cb067a76761d3355 but it's missing a tag, I think.

I opened https://github.com/ImageOptim/gifski/issues/163 to ask for it, once we have it we can remove this block.